src/components/auth/login.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react'
+import { navigate } from 'gatsby'
+import { signInWithEmailAndPassword } from 'firebase/auth'
+import {
+    AuthButtons,
+    AuthInput,
+    LoginForm,
+    LoginButton,
+    RegisterButton,
+    ResetButton,
+} from './styles'
+import { auth } from './firebase'
+import { Header } from 'components/elements'
+
+type TAuthLogin = {
+    setStatus: (status: string) => void
+}
+
+const AuthLogin = ({ setStatus }: TAuthLogin) => {
+    const [email, setEmail] = useState('')
+    const [password, setPassword] = useState('')
+    const [error, setError] = useState('')
+
+    const handleSubmit = async (e) => {
+        e.preventDefault()
+        try {
+            await signInWithEmailAndPassword(auth, email, password)
+            navigate(`/`)
+        } catch (err) {
+            setError(err.message)
+        }
+    }
+
+    return (
+        <div>
+            {error && <div></div>}
```

Comment:
what is inside this <div>?

Reply:
Actually nothing 😄 Made it just empt tag <></>